### ISSUE TITLE
Include more plugins

### ISSUE THREAD

**Issue by lucasvieites** [geany/geany-osx#17]
Hi there,
I'm not sure I'm in the right place, but I haven't found a better one.
I'd like to now how to request that a specific plugin be included in the osx package. I've tried compiling it myself, but I lack the permissions on my company-issued laptop.
The specific plugin I'd like to have is [Markdown](https://plugins.geany.org/markdown.html), for which I now have to use Atom.
I hope that you can help me with this.

Thank you in advance,
Lucas Vieites

**Comment by techee** [comment]
The problem is that the Markdown plugin uses WebKitGtk which doesn't seem to compile on macOS.

**Comment by codebrainz** [comment]
It's available for [macports](https://ports.macports.org/port/webkit-gtk3/summary/), though it doesn't look [straightforward to compile](https://github.com/macports/macports-ports/tree/master/www/webkit-gtk).

**Comment by lucasvieites** [comment]
@techee Thank you for your quick reply. I wasn't aware that this component wasn't available on macos.
@codebrainz Indeed, I did a little research after @techee replied, and it looks like a dependency hell to compile.

I've been thinking of setting up a build command for .md files and run that from Geany, then preview it in a browser.
Thanks again,
Lucas Vieites

**Comment by techee** [comment]
Yeah, I haven't been using macports but gtk-osx where I basically build everything by myself (with the necessary patches from gtk-osx). But it's not that well maintained and maybe I should give macports a try and investigate how I could create an app bundle using it.
